roles/node/defaults/main.yml

```diff
@@ -139,6 +139,14 @@ validator_stake_or_weight: 1
 ### In percent. Only used for the Primary Network
 validator_delegation_fee: 2
 
+## Optional systemd avalanchego.service parameters
+## Please refer to Ubuntu man pages for the usage of these parameters
+## https://manpages.ubuntu.com/manpages/jammy/en/man7/systemd.directives.7.html
+ExecStartPre: null
+ExecStartPost: null
+ExecStop: []
+ExecStopPost: null
```

Comment:
It would be more flexible and readable here to just accept a block of text
```suggestion
# Systemd
## Additional systemd avalanchego.service options
## Only modify this variable if you are absolutely sure about what you are doing
## Please refer to Ubuntu man pages for the usage of these options
## https://manpages.ubuntu.com/manpages/focal/en/man5/systemd.service.5.html
avalanchego_service_options: |
  # ExecStartPost=/bin/echo "Example ExecStartPost option"
```

Reply:
Also please respect the `snake_case` naming convention for variables!